readme.md

```diff
@@ -185,6 +185,7 @@ Thanks for contributing! 🦋🙌
 - [](# "file-finder-buffer") [Lets you start typing your search immediately after invoking the File Finder (<kbd>t</kbd>), instead of having you wait for it to load first.](https://user-images.githubusercontent.com/1402241/75542106-1c811700-5a5a-11ea-8aa5-bea0472c59e2.gif)
 - [](# "show-associated-branch-prs-on-fork") [Shows the associated pull requests on branches for forked repositories.](https://user-images.githubusercontent.com/16872793/81504659-7e5ec800-92b8-11ea-9ee6-924110e8cca1.png)
 - [](# "html-preview-link") [Adds a link to preview HTML files.](https://user-images.githubusercontent.com/44045911/67634792-48995980-f8fb-11e9-8b6a-7b57d5b12a2f.png)
+- [](# "file-age-colors") [Add wording that needs to be fixed](https://user-images.githubusercontent.com/16872793/178278788-277724fd-530b-420f-a3d0-d3597d7acfe8.png)
```

Comment:
Any suggestions for a better name?

Reply:
We could be more specific using "heatmap" but colors is easier to understand and find. Suggestions still welcome

Reply:
I think we should leave it as is and change it when we add it to other places.

@refined-github/maintainers any one have any suggestions?

Reply:
> I think we should leave it as is and change it when we add it to other places.

Renaming features has a cost (it breaks [the search](https://github.com/refined-github/refined-github/blob/844d8742c6af99ced0e6ae6a1b21618273b13bee/source/features/rgh-feature-descriptions.tsx#L20-L21) after every rename) so I've been trying not to.

I thought about it and I'm ok with having multiple features that do similar things in different areas, depending on how much they differ. We already share specific functions between features.